src/hook/useModites.tsx

```diff
@@ -21,36 +21,36 @@ const useModites = (records: Modite[]) => {
       sort_records = sort_records.reverse()
       break
     case 'firsta':
-      sort_records.sort((a, b) => {
+      sort_records.sort((a: any, b: any) => {
         const aFirst = a.real_name.split(' ').shift(),
           bFirst = b.real_name.split(' ').shift()
         return aFirst.localeCompare(bFirst)
       })
       break
     case 'firstd':
-      sort_records.sort((a, b) => {
+      sort_records.sort((a: any, b: any) => {
         const aFirst = a.real_name.split(' ').shift(),
           bFirst = b.real_name.split(' ').shift()
         return bFirst.localeCompare(aFirst)
       })
       break
     case 'tacosa':
-      sort_records.sort((a, b) => {
+      sort_records.sort((a: any, b: any) => {
         return a.tacos - a.tacos
```

Comment:
there's an error here, ahould be `a - b`, i fixed this in https://github.com/moduslabs/modir/pull/100 but incase we are not merging that please can you include it in this PR ?

Reply:
yes sure, i think we need to sync about these pr's.